docs/get-started.asciidoc

```diff
@@ -1,23 +1,172 @@
+////
+Goal of this doc:
+The user is able to successfully see data from their Java application make it to the Elastic UI via the Elastic Distribution for OpenTelemetry Java
+
+Assumptions we're comfortable making about the reader:
+* They are familiar with Elastic
+* They are familiar with OpenTelemetry
+* They have Java installed
+////
+
 [[get-started]]
 == Get started
 
-This guide shows you how to use the Elastic Distribution for OpenTelemetry Java to instrument your Java application and send OpenTelemetry data to an Elastic Observability deployment.
+:language: Java
+:language_lc: java
+:distro_name: Elastic Distribution for OpenTelemetry {language}
+
+include::release-status.asciidoc[]
+
+This guide shows you how to use the Elastic Distribution for OpenTelemetry {language} ("the distro") to instrument your {language} application and send OpenTelemetry data to an Elastic Observability deployment.
```

Comment:
Do we need to explain "the distro" on every page?

Reply:
I like to establish the alias on every page because a user could land on any of the pages first (especially via search results... every page is page one), but I don't think we have a hard rule for Elastic docs.